.ci/gcb-push-downstream.yml

```
@@ -104,20 +126,11 @@ steps:
             fi
 
     # TGC
-    - name: 'gcr.io/graphite-docker-images/bash-plus'
-      entrypoint: '/workspace/.ci/scripts/bash-plus/downstream-waiter/wait_for_commit.sh'
-      id: tgc-sync
-      waitFor: ["checkout"]
-      args:
-          - 'tgc-sync'
-          - $BRANCH_NAME
-          - $COMMIT_SHA
-
     - name: 'gcr.io/graphite-docker-images/build-environment'
       entrypoint: '/workspace/.ci/scripts/go-plus/magician/exec.sh'
       secretEnv: ["GITHUB_TOKEN_CLASSIC"]
       id: tgc-push
-      waitFor: ["tgc-sync", "tpgb-push"]
+      waitFor: ["tpg-sync", "tpgb-sync", "tgc-sync", "tf-oics-sync", "tpgb-push"]
```

tpgb-push is already waiting on the other items now.

```suggestion
      waitFor: ["tpgb-push"]
```